api/v3/Contact.php

```diff
@@ -376,6 +376,30 @@ function _civicrm_api3_contact_get_supportanomalies(&$params, &$options) {
     unset($params['filter.group_id']);
     $options['input_params']['group'] = $groups;
   }
+  if (isset($params['group'])) {
+    $groups = $params['group'];
+    $allGroups = CRM_Core_PseudoConstant::group();
+    if (is_array($groups) && in_array(key($groups), CRM_Core_DAO::acceptedSQLOperators(), TRUE)) {
+      $groupsArray = $groups[key($groups)];
+      foreach ($groupsArray as $group => &$ignore) {
+        if (!is_numeric($group) && array_search($group, $allGroups)) {
+          $group = array_search($group, $allGroups);
+        }
+      }
+      $groups[key($groups)] = $groupsArray;
+    }
+    elseif (is_array($groups)) {
+      foreach ($groups as $k => &$group) {
+        if (array_search($group, $allGroups)) {
+          $group = array_search($group, $allGroups);
```

Comment:
when you say @eileenmcnaughton when you say  this not being used what do you mean?